src/math/Color.js

```diff
@@ -580,6 +580,16 @@ Object.assign( Color.prototype, {
 
 	},
 
+	fromBufferAttribute: function ( attribute, index ) {
```

Comment:
The attribute could normalized, and the attribute array a `Uint8Array`, as in https://threejs.org/examples/webgl_buffergeometry_points_interleaved.html. I expect that would be a good test case.

Reply:
I've added a check for normalized data. However, `webgl_buffergeometry_points_interleaved` is not a good test case since `Geometry.fromBufferGeometry()` only supports mesh geometries (see #14022).